plugin-maven/README.md

```diff
@@ -299,4 +299,34 @@ By default, `spotless:check` is bound to the `verify` phase.  You might want to
 
 TODO
 
+<a name="custom_eclipse"></a>
+
+## Custom Eclipse versions
+
+As described in the corresponding language specific sections, for Spotless Eclipse formatters multiple versions are supported. Not all Eclipse versions are supported, since not all Eclipse version changes have a direct impact on the formatting.
+
+For example the formatter versions for Java Eclipse are listed [here](https://github.com/diffplug/spotless/tree/master/lib-extra/src/main/resources/com/diffplug/spotless/extra/config/eclipse_jdt_formatter). For each formatter version a file is defined listing all its dependencies with a fixed version. For example:
+
+```
+...
+org.eclipse.jdt:org.eclipse.jdt.core:3.12.0
+org.eclipse.platform:org.eclipse.core.resources:3.11.0
+...
+```
+
+In case you are missing a latest change, you can override the default dependencies. For example:
+
+```xml
+...
+     <eclipse>
+       <file>${basedir}/eclipse-fmt.xml</file>
+       <dependencies>
+         <param>org.eclipse.jdt:org.eclipse.jdt.core:3.13.2</param>
```

Comment:
Can it be `<dependency>...</dependency>` instead of `<param>...</param>`? "Dependency" would read a bit nicer

Reply:
Honestly, I hoped that you can tell me how Maven Plugin treats Array array parameters.
The code is:
```
	@Parameter
	private String[] dependencies;
```
Can I just use any XML element name for the single values I like? Did not find this obvious in the Mavan documentation, but I had no time to read it in detail. I would very much appreciate if you could change the `Eclipse.java` so that it works.
BTW: I removed the `required = true` from the `file` parameter. From Eclipse formatter / configuration side, you don't have to specify it, but I am not sure whether we already discussed that and you found it cleaner to make it as required. Can you remind me?

Reply:
I think maven does not really care about names of the nested tags :) My comment was just about the README file. Doc I used for the maven plugin is [here](https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Mapping_Lists). Quick local experiment shows that all the following configs result in `[1, 2, 3]`:

```xml
<dependencies>
  <dependency>1</dependency>
  <dependency>2</dependency>
  <dependency>3</dependency>
</dependencies>

<dependencies>
  <param>1</param>
  <param>2</param>
  <param>3</param>
</dependencies>

<dependencies>
  <dependency>1</dependency>
  <param>2</param>
  <foo>3</foo>
</dependencies>
```

We have not discussed the `file` parameter. It is definitely okay for it to be optional if Eclipse formatter can handle this.